.github/workflows/notify.yml

```diff
@@ -0,0 +1,25 @@
+name: Notify about failed build
+
+on:
+  workflow_run:
+    workflows: ["Tests"]
+    types: [completed]
+
+jobs:
+  notify:
+    name: 📣 Notify community on failure
+    if: ${{  github.event_name == 'schedule' && github.event.workflow_run.conclusion == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send email on failure
+        if: ${{ github.event_name == 'schedule' && github.event.workflow_run.conclusion == 'failure' }}
+        uses: dawidd6/action-send-mail@v2
+        with:
+          server_address: ${{ secrets.SMTP_SERVER }}
+          server_port: ${{ secrets.SMTP_PORT }}
+          username: ${{ secrets.SMTP_USERNAME }}
+          password: ${{ secrets.SMTP_PASSWORD }}
+          subject: GitHub Actions failed for ${{ github.repository }}
+          body: Build job of ${{ github.repository }} failed! See https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }} for more information.
+          to: ${{ secrets.MAIL_RECEIVER }}
```

Comment:
If we do want this, someone who has admin access to this repository will need to enter these secrets. The workflow definition is copied pretty much 1:1 from another repository, so I don't expect hard-to-debug failures. I have already some working credentials available, please contact me privately and I'll send them to you. :D

Reply:
To which addresses would you like to send this? I guess most of the folks on squeak-dev are not interested in FileSystem. And neither is Pharo in this different version of the library.

Reply:
My first guess would have been squeak-dev since this repository is part of @squeak-smalltalk. Alternatively, I can also add me and you and some other interested people ...

Reply:
The ston, tonel, and filesystem repositories are a bit odd in this organization. They are effectively not as supported as, say, the Trunk of Squeak is. I do not believe that either of the three would be undisputedly considered to be part of Squeak, as opposed to what is in Trunk.